data/strings/strings.txt

```diff
@@ -20808,7 +20808,7 @@
 
   [mobile_data_option_not_today]
     tags = android,ios
-    en = Do not Use Today
+    en = Do Not Use Today
```

Comment:
this falls under the capitalization debate also

Reply:
I don't think so, it's written "Do Not Disturb" not "Do not Disturb"

Reply:
I couldn't find anywhere that says you shouldn't capitalize "not"

https://titlecapitalize.com/is-not-capitalized-in-a-title/

Reply:
+1 `Not`